src/index.js

```diff
@@ -53,8 +53,7 @@ function EvolvClient(opts) {
     clientName: options.clientName
   };
 
-  const contextBeacon = options.analytics ? new Beacon(options.endpoint + '/' + options.environment + '/data', context, beaconOptions) : null;
-  const eventBeacon = options.beacon || new Beacon(options.endpoint + '/' + options.environment + '/events', context, beaconOptions);
```

Comment:
Was the idea that event beacon would be used if analytics was turned off, but now we only use analytics so it'll always be made?

Reply:
contextBeacon came in after eventBeacon. eventBeacon gave all the info for visitor based tests. Session based were new and were set to be optional. Now all tests are session based